lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -378,6 +384,71 @@ def __str__(self):
             self.distribution, self.desired_units,
             self.predictor_of_mean_flag, self.minimiser)
 
+    def create_coefficients_cube(
+            self, optimised_coeffs, current_forecast):
+        """Create a cube for storing the coefficients computed using EMOS.
+
+        .. See the documentation for examples of these cubes.
+        .. include:: extended_documentation/ensemble_calibration/
+           ensemble_calibration/create_coefficients_cube.rst
+
+        Args:
+            optimised_coeffs (list):
+                List of optimised coefficients.
+                Order of coefficients is [gamma, delta, alpha, beta].
+            current_forecast (iris.cube.Cube):
+                The cube containing the current forecast.
+
+        Returns:
+            cube (iris.cube.Cube):
+                Cube constructed using the coefficients provided and using
+                metadata from the current_forecast cube. The cube contains
+                a coefficient_index dimension coordinate where the points
+                of the coordinate are integer values and a
+                coefficient_name auxiliary coordinate where the points of
+                the coordinate are e.g. gamma, delta, alpha, beta.
+
+        """
+        if self.predictor_of_mean_flag.lower() in ["realizations"]:
```

Comment:
Why is this test done as an A in B type? Is this to capture "realization" and "realizations"?

Reply:
I've updated this to the more standard: 
```
if self.predictor_of_mean_flag.lower() == "realizations":
```